site/js/modules/webui/render.js

```diff
@@ -64,6 +64,157 @@ function makeCardSection(sectionName, sectionCards) {
     `;
 }
 
+// Render a page that follows a standard template.
+// The template includes a control area, header, description, input area, submission button, and a results area.
+// The onSubmitFunc must do the following:
+//   - Accept four parameters (params, context, container, inputParams)
+//   - Return a promise that resolves to the content to display in the results area
```

Comment:
I would end both of these items with a period (since they are complete sentences).